jujugui/static/gui/src/app/components/entity-details/content/test-content.js

```diff
@@ -31,8 +31,22 @@ function _generateTagItem(tag, fn) {
   ];
 }
 
+function generateScript(isBundle) {
+  let id = 'trusty/django-123';
+  if (isBundle) {
+    id = 'django-cluster';
+  }
+
+  return '<script ' +
+  'src="https://assets.ubuntu.com/v1/juju-cards-v1.5.0.js"></script>\n' +
+  '<div class="juju-card" data-id="'+id+'"></div>';
```

Comment:
I initially used template literals, but linting complained about long lines. So then you add newlines, and the newlines are included in the output.